controllers/nodehealthcheck_controller.go

```diff
@@ -54,19 +54,20 @@ import (
 )
 
 const (
-	oldRemediationCRAnnotationKey    = "nodehealthcheck.medik8s.io/old-remediation-cr-flag"
-	remediationTimedOutAnnotationkey = "remediation.medik8s.io/nhc-timed-out"
-	remediationCRAlertTimeout        = time.Hour * 48
-	eventReasonRemediationCreated    = "RemediationCreated"
-	eventReasonRemediationSkipped    = "RemediationSkipped"
-	eventReasonRemediationRemoved    = "RemediationRemoved"
-	eventReasonNoTemplateLeft        = "NoTemplateLeft"
-	eventReasonDisabled              = "Disabled"
-	eventReasonEnabled               = "Enabled"
-	eventTypeNormal                  = "Normal"
-	eventTypeWarning                 = "Warning"
-	enabledMessage                   = "No issues found, NodeHealthCheck is enabled."
-	conditionTypeSucceeded           = "Succeeded"
+	oldRemediationCRAnnotationKey        = "nodehealthcheck.medik8s.io/old-remediation-cr-flag"
+	remediationTimedOutAnnotationkey     = "remediation.medik8s.io/nhc-timed-out"
+	remediationUnhealthyReasonAnnotation = "remediation.medik8s.io/unhealthy-reason"
```

Comment:
Maybe `unhealthy-node` which is more informative rather than `unhealthy-reason`?

Reply:
I'm open to new ideas for this name, but this annotation is supposed to get the _reason_ why the node was considered unhealthy.

see https://github.com/medik8s/node-healthcheck-operator/blob/e7fb9fab083d41fb756e9435976b147441563619/controllers/nodehealthcheck_controller.go#L434

maybe `unhealthy-node-reason`?

Reply:
Yes, sounds better. Also the const name should change if we go this suggestion